docker/python/Dockerfile

```diff
@@ -36,3 +36,21 @@ RUN apt-get update \
 ENV PYTHON_PIP_VERSION 10.0.1
 RUN wget --no-check-certificate -O /tmp/get-pip.py 'https://bootstrap.pypa.io/get-pip.py' \
     && python3 /tmp/get-pip.py
+
+# Create a file that only root can access. This file is tested in
+# `tests/python/test_sudo.py`.
+
+RUN echo -n 'words of wisdom' > /etc/secret_file.txt \
+  && chown root.root /etc/secret_file.txt \
+  && chmod 0600 /etc/secret_file.txt
+
+# Create a user and allow sudo
+ARG UID
+ARG USERNAME
```

Comment:
nit: should `ARG` options be at the top of the `Dockerfile`? Also, does this have a default value?

Reply:
If we put the ARGs at the top, we can not utilize the cache as much as we do now (the cache will be invalidated after the ARG if we use different ARGs).

They could have default value if it's needed, but we don't need it in this file.